src/pinball_wizard.coffee

```diff
@@ -120,6 +120,31 @@ define ->
     method = params.shift()
     @[method].apply(@, params)
 
+
+  storage = window.localStorage
+
+  setPermanent = (value) ->
+    storage.setItem('pinball_wizard', JSON.stringify(value))
+
+  appendPermanent = (name) ->
+    l = permanent()
+    l.push(name)
+    setPermanent l
+
+  permanent = ->
+    JSON.parse(storage.getItem('pinball_wizard') or "[]") or []
+
+  activatePermanently = (name) ->
+    appendPermanent(name)
+    activate(name, 'permanent')
+
+  resetPermanent = ->
+    setPermanent []
+
+  activateAllPermanent = ->
```

Comment:
Should we prefix this with an underscore to indicate that it's "private"? Probably doesn't need to be part of the API, right?